coordinator.go

```diff
@@ -44,6 +44,9 @@ type Coordinator interface {
 	//
 	// Do not release tasks. The consumer will handle task releasing.
 	Close()
+
+	// String should return a name for the coordinator to use in logging.
+	String() string
```

Comment:
Since the comment states it used to log the name, should it just be `Name()` or `NodeId()`.